base_layer/wallet/src/output_manager_service/master_key_manager.rs

```diff
@@ -41,14 +43,32 @@ use crate::{
 };
 
 const LOG_TARGET: &str = "wallet::output_manager_service::master_key_manager";
-
-const KEY_MANAGER_COINBASE_BRANCH_KEY: &str = "coinbase";
-const KEY_MANAGER_COINBASE_SCRIPT_BRANCH_KEY: &str = "coinbase_script";
-const KEY_MANAGER_SCRIPT_BRANCH_KEY: &str = "script";
-const KEY_MANAGER_RECOVERY_VIEWONLY_BRANCH_KEY: &str = "recovery_viewonly";
-const KEY_MANAGER_RECOVERY_BLINDING_BRANCH_KEY: &str = "recovery_blinding";
 const KEY_MANAGER_MAX_SEARCH_DEPTH: u64 = 1_000_000;
 
+#[derive(Clone, Copy)]
+pub enum KeyManagerBranch {
+    Spend,
+    SpendScript,
+    Coinbase,
+    CoinbaseScript,
+    RecoveryViewOnly,
+    RecoveryBlinding,
+}
+
+impl Display for KeyManagerBranch {
+    fn fmt(&self, fmt: &mut Formatter<'_>) -> Result<(), Error> {
+        let response = match self {
+            KeyManagerBranch::Spend => "",
```

Comment:
Should this be blank?

Reply:
Ja, in the beginning the first branch key was just blank but now it would be a breaking change to update that :P